### PR TITLE
Migrated Account Functions to use Js SDK

### PR DESCRIPTION
**Pull request by tomijaga**
#585 

https://github.com/thenewboston-developers/thenewboston-js/issues/98
https://github.com/thenewboston-developers/thenewboston-js/issues/99
https://github.com/thenewboston-developers/thenewboston-js/issues/100
https://github.com/thenewboston-developers/thenewboston-js/issues/101
https://github.com/thenewboston-developers/thenewboston-js/issues/102

### CODE REVIEW

**Comment by tomijaga** [comment]
Hey, @angle943 how do you want to review the code?
Do u want to review and merge each change or could you give me access to merge then you could review the code all at once when I'm done

**Comment by zinoadidi** [comment]
@angle943 maybe you can assign one of the other contributors/team members to review for us, then you only need to check it when we are ready to merge to development

**Comment by angle943** [comment]
@zinoadidi i gave you write permissions in this repo. After you've reviewed, feel free to merge it to the `javascript-sdk-migration` branch. Once that branch is fully ready, i'll review it

File: src/renderer/utils/blocks.ts [hunk 1]
```diff
@@ -78,8 +67,9 @@ export const sendBlock = async (
 
   const {ip_address: ipAddress, port, protocol} = activeBankConfig;
   const address = formatAddress(ipAddress, port, protocol);
-
-  const block = await createBlock(primaryValidatorConfig, senderSigningKey, senderAccountNumber, txs);
+  const sender = new Account(senderSigningKey);
+  const balanceLock = await fetchAccountBalanceLock(senderAccountNumber, primaryValidatorConfig);
+  const block = sender.createBlockMessage(balanceLock, txs);
   await axios.post(`${address}/blocks`, block, {
```
Comment (zinoadidi) [inline src/renderer/utils/blocks.ts:73]: The post requests here will be handled by sdk in later fix right ?
Or how is it duppose to work; I believe all interactions with chain will go through us eventually

Comment (tomijaga) [reply src/renderer/utils/blocks.ts:73]: It would be handled when I implement the payment handler class

**Review by zinoadidi — APPROVED** [review]

**Comment by zinoadidi** [comment]
🥳

**Pull request merged**